src/components/users/subscriptions/subscriptionsHandlers.ts

```diff
@@ -14,4 +16,19 @@ export class SubscriptionsHandlers implements IWidgetHandler {
 
         return widgetOrder;
     }
+
+    public getStyleDefinitions(): StyleDefinition {
+        return {
+            components: {
+                userSubscriptions: {
+                    displayName: "User: Subscriptions",
+                    plugins: ["typography", "background", "margin", "padding"],
+                    components: {
+                        subscriptionsGridHeader: DefaultStyleDefinitions.GridHeader,
```

Comment:
Is it possible to use `tableRow` name here to just override the base class instead of introducing `subscriptionsGridRow`?